x-pack/plugins/enterprise_search/public/applications/app_search/components/synonyms/components/empty_state.test.tsx

```diff
@@ -21,7 +23,7 @@ describe('EmptyState', () => {
 
     expect(wrapper.find('h2').text()).toEqual('Create your first synonym set');
     expect(wrapper.find(EuiButton).prop('href')).toEqual(
-      expect.stringContaining('/synonyms-guide.html')
+      expect.stringContaining(`${docLinks.appSearchSynonyms}`)
```

Comment:
```suggestion
      expect.stringContaining(docLinks.appSearchSynonyms)
```
No need for a template literal here